index.html

```diff
@@ -282,9 +282,9 @@ <h5 class="partners-title">Partner</h5>
         <!-- footer section -->
 
         <section class="footer-section container footer-content d-flex justify-content-center">
-                <div class="footer-logo">
+                <h5 class="footer-logo">
                     GCRECC
-                </div>
```

Comment:
In the `script.js` file.

- Kindly make sure that the section "Featured speakers" is generated dynamically with JavaScript. Please apply the knowledge that you gained when creating the portfolio during the HTML/CSS module to achieve this. You should not have any hardcoded information about the "Featured speakers" on the `index.html` file. The should be a function that iterates through the speakers list on window load and creates HTML elements with the data from each object.

- Kindly create an array or object for the list of speakers used to generate the HTML. The list of speakers should contain a speaker object for each that holds their specific data. e.g
```
const speakerList = [ { name: "Speaker Name",... } ]
```

- The object should contain all the information needed to generate the dynamic element, at least the name, role, description, and image URL of the speaker. For example:
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```